### PR TITLE
FITWD-103:Added condition when title value is empty.

### PR DESCRIPTION
**Pull request by shweta038**

### CODE REVIEW

File: src/Components/ParagraphDisplay/ParagraphDisplayTiles.tsx [hunk 1]
```diff
@@ -204,16 +204,17 @@ export class ParagraphDisplayTiles extends React.Component<
...
+            <div className="col">
+              <h2> {data.field_title} </h2>
+            </div>
+          )}</>
         )
         : (data.field_view_mode == "card-large")
           ? (
             <div className="col-12 col-lg-3 subheader">
-              <h2> {data.field_title} </h2>
+              <h2 {data.field_title} </h2>
```
Comment (uditrwt) [inline src/Components/ParagraphDisplay/ParagraphDisplayTiles.tsx:217]: @shweta038 , Can you please correct the h2 tag.

**Comment by milkenwebdev01** [comment]
Visual regression test failed! [Visual report](https://circleci.com/gh/milken-institute/FreshDrupalMI/12387/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by milkenwebdev01** [comment]
Visual regression test failed! [Visual report](https://circleci.com/gh/milken-institute/FreshDrupalMI/12388/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by milkenwebdev01** [comment]
Visual regression test failed! [Visual report](https://circleci.com/gh/milken-institute/FreshDrupalMI/12389/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by milkenwebdev01** [comment]
Visual regression test failed! [Visual report](https://circleci.com/gh/milken-institute/FreshDrupalMI/12390/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by milkenwebdev01** [comment]
Visual regression test failed! [Visual report](https://circleci.com/gh/milken-institute/FreshDrupalMI/12391/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Comment by milkenwebdev01** [comment]
Visual regression test failed! [Visual report](https://circleci.com/gh/milken-institute/FreshDrupalMI/12392/artifacts/0/artifacts/backstop_data/html_report/index.html)

**Pull request merged**